perun/logic/store.py

```diff
@@ -101,6 +101,19 @@ def is_sha1(checksum):
     return len(checksum) == 40 and all(c in string.hexdigits for c in checksum)
 
 
+def sha_path_to_sha(sha_path):
```

Comment:
I'm rolling on floor laughing, this sounds like some magician's spell.

Reply:
Renamed to a 'version_path_to_sha'.